pkg/common/keys.go

```diff
@@ -176,8 +176,12 @@ const (
 	// to used for the argocd container.
 	ArgoCDImageEnvName = "ARGOCD_IMAGE"
 
+	// ArgoCDRedisHAProxyImageEnvName is the environment variable used to get the image
+	// to used for the Redis HA Proxy container.
+	ArgoCDRedisHAProxyImageEnvName = "ARGOCD_REDIS_HA_PROXY_IMAGE"
```

Comment:
Please update the doc in dcos/reference/argcd.md

Reply:
done